training/train.py

```diff
@@ -15,8 +15,10 @@
 
 overfit = args.overfit
 
+Image_set = 'train'
+
 dataset_val = torchvision.datasets.VOCDetection(
-    root="../VOC", year="2007", image_set="train", download=False
+    root="../VOC", year="2007", image_set=Image_set, download=True
 )
```

Comment:
Please followe the PEP8 standards for naming https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
done